constants/error-constants.js

```diff
@@ -112,6 +112,7 @@ const ERROR = {
     // delete submission error file list
     DELETE_NO_FILE_SUBMISSION: "No extra file found",
     DELETE_NO_DATA_FILE_EXISTS: "No data files found for deletion in the bucket",
+    INVALID_DELETE_SUBMISSION_STATUS : "Requested submission is not allowed to delete",
```

Comment:
Let's update this message to "Data Submissions that have been released cannot be deleted"

Reply:
pushed it for the update.